### PR TITLE
More ldp fixes

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: src/lib/operations/readBlob.ts [hunk 1]
```diff
@@ -16,6 +19,10 @@ export async function readBlob (task: WacLdpTask, blob: Blob): Promise<WacLdpRes
     return result
   }
   result.resourceData = await streamToObject(await blob.getData())
+  if (task.asJsonLd) {
+    const rdf = resourceDataToRdf(result.resourceData)
+    result.resourceData = await rdfToResourceData(rdf, true)
+  }
```
Comment (RubenVerborgh) [inline src/lib/operations/readBlob.ts]: Maybe add TODO for conneg?

**Comment by michielbdejong** [comment]
Also, r? @sylvainlb

File: src/lib/operations/readBlob.ts (hunk 1 above)
Comment (michielbdejong) [reply src/lib/operations/readBlob.ts]: done in #71

**Comment by michielbdejong** [comment]
All changes from this PR were included in #71.

**Pull request closed without merging**